scene/animation/animation_player.h

```diff
@@ -315,6 +315,8 @@ class AnimationPlayer : public Node {
 
 	static void _bind_methods();
 
+	GDVIRTUAL5RC(Variant, _post_process_key_value, const Ref<Animation> &, int, Variant, const Object *, int);
```

Comment:
I think constness gets lost in GDVIRTUAL
```suggestion
	GDVIRTUAL5RC(Variant, _post_process_key_value, Ref<Animation>, int, Variant, Object *, int);
```